server/controllers/list.js

```diff
@@ -21,25 +21,27 @@ const {
   responseWithListMembers
 } = require('../common/utils/index');
 const { updateSubdocumentFields } = require('../common/utils/helpers');
+const { ListType } = require('../common/variables');
 
 const createList = (req, resp) => {
-  const { description, isListPrivate, name, cohortId } = req.body;
+  const { cohortId, description, name, type } = req.body;
   const {
     user: { _id: userId }
   } = req;
 
   const list = new List({
     cohortId,
     description,
-    isPrivate: isListPrivate,
     memberIds: userId,
     name,
     ownerIds: userId,
+    type,
     viewersIds: userId
   });
 
-  if (cohortId && !isListPrivate) {
+  if (cohortId && type === ListType.SHARED) {
```

Comment:
I think it would be better to store this in a `const isListShared = type === ListType.SHARED`